# Exercise 15.1c solution

Using $V(x) = V_c \, e^{-x/\lambda_{\mathrm{m}}}$ with $V_c = 10$ mV and $\lambda_{\mathrm{m}} \approx 0.108$ cm (1.08 mm):


In [2]:
import numpy as np

# Recompute lambda_m from 15.1b
R_m = 7000  # Ohm cm^2
R_c = 150  # Ohm cm
d = 10e-4  # cm
lambda_m = np.sqrt(R_m * d / (4 * R_c))

V_c = 10  # mV

# 1) Voltage at 1 mm = 0.1 cm
x1 = 0.1  # cm
V_1mm = V_c * np.exp(-x1 / lambda_m)
print(f"lambda_m = {lambda_m:.4f} cm = {lambda_m * 10:.2f} mm")
print(f"1) Voltage at 1 mm: {V_1mm:.2f} mV")

# 2) Voltage at 1 cm
x2 = 1.0  # cm
V_1cm = V_c * np.exp(-x2 / lambda_m)
print(f"2) Voltage at 1 cm: {V_1cm:.2e} mV (essentially zero)")

# 3) Voltage at 0.5 mm = 0.05 cm
x3 = 0.05  # cm
V_05mm = V_c * np.exp(-x3 / lambda_m)
print(f"3) Voltage at 0.5 mm (soma): {V_05mm:.2f} mV")

lambda_m = 0.1080 cm = 1.08 mm
1) Voltage at 1 mm: 3.96 mV
2) Voltage at 1 cm: 9.53e-04 mV (essentially zero)
3) Voltage at 0.5 mm (soma): 6.29 mV


A single synapse producing 10 mV would deliver ~6.3 mV to the soma. This is below the ~15 mV threshold, so multiple synaptic inputs must summate (spatial and temporal summation) to trigger an AP.
